packages/typeorm-tailordb-codegen/package.json

```diff
@@ -12,40 +12,28 @@
     "typeorm-tailordb-codegen": "./dist/cli.js"
   },
   "files": [
-    "dist"
+    "dist",
+    "CHANGELOG.md"
   ],
   "scripts": {
     "build": "node ./build.js",
     "check": "biome check .",
     "check-write": "biome check --write .",
-    "type-check": "tsc"
+    "type-check": "tsc",
+    "prepublishOnly": "pnpm run build"
   },
   "dependencies": {
-    "@tailor-platform/function-typeorm-tailordb-codegen": "link:",
```

Comment:
I removed `link:` since it causes errors when installing with npm as it can't be interpreted. This should be unnecessary, right...?